doc/ref_model/chapters/appendix-a.md

```diff
@@ -50,6 +102,8 @@ Current CNTT Policies:
 
 **SR-IOV:**
 
+Note on CNTT VNF Transition Guidelines plan for SR-IOV with vendor-specific drivers in VNFs: Currently (end of 2019) SR-IOV is often used as most performant way of getting packets into VNFs with tradeoffs in agility (VNF onboarding), operational issues (vendor drivers in VNFs) and changes in networking and security management. The tradeoff of allowing SR-IOV Virtual Function PCI Pass Through for VNF network connectivity is acceptable only until a solution is available without vendor-specific driver and acceptable resource consumption (CPU). As other connectivity technologies with open standard interfaces improve both technically and commercially, future CNTT releases will phase out SR-IOV with vendor-specific drivers in VNFs and any other connectivity technology tightly coupling applications and infrastructure.
```

Comment:
This paragraph and the following one are substantially about the same thing.  If the following paragraph is modified, what if any elements of this paragraph are not covered and should remain?

Reply:
Indeed, it is similar. The only difference is that it tries to summarize the tradeoff (_"... often used as most performant way of getting packets into VNFs with tradeoffs in agility (VNF onboarding), operational issues (vendor drivers in VNFs) and changes in networking and security management._"). Later text lists detailed "anecdotes". Those could be combined to have first the summary of tradeoff followed by detailed "anecdotes". Should I do that?

Reply:
> Those could be combined to have first the summary of tradeoff followed by detailed "anecdotes". Should I do that?

yes, please.

Reply:
Done.